SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -1845,14 +1846,14 @@
                 "qpdf",
                 "qperf",
                 "qr-code-generator",
-                "qt5-qtbase",
+                "qt-rpm-macros",
                 "qt5-qtconnectivity",
```

Comment:
These still have qt5 in them.  Is that intentional?

Reply:
the packages which still have "qt5-" in them are all extended packages which have not been updated as a part of this work. Should these also be upgraded with this PR?